zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectsDto.kt

```diff
@@ -0,0 +1,6 @@
+package com.ritense.objectmanagement.domain
+
+data class ObjectsDto(
+    val key: String,
+    val recordIndex: Int
```

Comment:
This dto isn't correct yet. This Dto should be similar to:
https://github.com/valtimo-platform/valtimo-backend-libraries/blob/development/case/src/main/kotlin/com/ritense/case/web/rest/dto/CaseListRowDto.kt

An example of what the response of the endpoint should look like can be found here:
https://ritense.atlassian.net/wiki/spaces/RIT/pages/1107296265/Object+management+-+API+specification#Get-object-list

Reply:
I see in the docs the key URL and in the DTO I see the the key ID. I need a single uuid to target the detail page so either add id and url or just the id (maybe the docs is outdated then I said nothing).

Reply:
Hi Kevin, I'm afraid the BE needs the entire objectUrl. So that the BE knows which objectenAPI to use when retrieving the object. (There can be multiple objectenAPI's)

Also, right now, there is no endpoint for retrieving the object details. That endpoint will be added in this story:
https://ritense.tpondemand.com/entity/48901-be-api-endpoint-for-retrieving-prefilled